packages/swirl-components/src/components/swirl-card/swirl-card.css

```diff
@@ -13,7 +13,7 @@
   margin: 0;
   padding: 0;
   border: var(--s-border-width-default) solid var(--s-border-default);
-  border-radius: var(--s-border-radius-base);
+  border-radius: var(--s-border-radius-xl);
```

Comment:
Yes, it was decided to use the `xl` border radius on both posts as well as cards in the web app. It's one of the changes done for the highlights feature. https://flipapp.myjetbrains.com/youtrack/agiles/115-8/current?issue=APP-14300

Reply:
Ah nvm. Did not look into the description at first, because the PR didn't have the name of the Ticket in it as usual with APP-[ticketnumber] and short PR description.
Then alright